.github/workflows/docker-release.yml

```diff
@@ -20,10 +21,51 @@ jobs:
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
-      - name: Get the tag name
+
+      - name: Get the new tag version
         run: |
           echo ${GITHUB_REF##*/}
-          echo "TAG=${GITHUB_REF##*/}" >> $GITHUB_ENV
+          echo "NEW_TAG_VERSION=${GITHUB_REF##*/}" >> $GITHUB_ENV
+
+      - name: Find commit for latest tag
+        run:  |
+          COMMIT=$(git rev-list -n 1 docker/latest)
+          echo "COMMIT_FOR_LATEST_TAG=${COMMIT}" >> $GITHUB_ENV
+
+      - name: Find tag version associated with latest tag
+        run: |
+          LATEST_TAG_VERSION=$(git tag --list "docker/v*" --points-at ${{env.COMMIT_FOR_LATEST_TAG}})
+          echo "LATEST_TAG_VERSION=${LATEST_TAG_VERSION}" >> $GITHUB_ENV
+
+      - name: Check Tag Versions
+        uses: actions/github-script@v5
+        id: set-result
+        env:
+          LATEST_TAG_VERSION: ${{env.LATEST_TAG_VERSION}}
+          NEW_TAG_VERSION: ${{env.NEW_TAG_VERSION}}
+        with:
+          script: |
+            const { LATEST_TAG_VERSION , NEW_TAG_VERSION } = process.env
+            
+            if (!LATEST_TAG_VERSION) {
+              return "true"
+            }
+
+            const latestTagVersion = LATEST_TAG_VERSION.split('docker/v')
+            const newTagVersion = NEW_TAG_VERSION.split('v')
+
+            const latestTagVersionArray = latestTagVersion[1].split('.')
+            const newTagVersionArray = newTagVersion[1].split('.')
+
+            const latestTagVersionNum = latestTagVersionArray.join("")
+            const newTagVersionNum = newTagVersionArray.join("")
+
+            if (newTagVersionNum > latestTagVersionNum) {
```

Comment:
Hmm... This will fail in cases of a double digit version number. `1.10.0` should be < `2.0.0`, but it isn't. I think the array entries must be tested independently for semantic versioning.